packages/babel-core/test/get-possible-preset-names.js

```diff
@@ -0,0 +1,22 @@
+let assert                 = require("assert");
+let getPossiblePresetNames = require("../lib/helpers/get-possible-preset-names");
+
+describe("getPossiblePresetNames", function () {
+  it("adds the babel-preset prefix", function() {
+    assert.deepEqual(getPossiblePresetNames("foobar"), ["babel-preset-foobar", "foobar"]);
+  });
+
+  it("inserts babel-preset after @org/", function() {
+    assert.deepEqual(getPossiblePresetNames("@babel/es2015"), [
```

Comment:
I agree, but in this PR I just wanted to preserve the existing behavior. I can remove that in a separate PR if there's consensus on it.